types/advantech/WISE2410/climate.schema.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$id": "https://akenza.io/advantech/wise-2410/climate.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "processingType": "uplink_decoder",
+  "topic": "climate",
+  "title": "Climate",
+  "properties": {
+    "temperature": {
+      "$ref": "https://raw.githubusercontent.com/akenza-io/device-type-library/main/data-models/environment/schema.json#/$defs/temperature/celsius"
```

Comment:
note that this should be changed based on the temperature unit, I suggest to always output temperature (C which is default), temperatureF and temperatureK